utils/notification_service.py

```diff
@@ -1426,7 +1426,17 @@ def pop_default(l: list[Any], i: int, default: Any) -> Any:
     if len(matrix_job_results) > 0:
         target_results = matrix_job_results
     else:
-        target_results = additional_results[job_to_test_map[job_name]]
+        default_result = {
+            "failed": {"unclassified": 0, "single": 0, "multi": 0},
+            "success": 0,
+            "time_spent": "",
+            "error": False,
+            "failures": {},
+            "job_link": {},
```

Comment:
Would there be a way to say this is the default result? Like using -1 instead of 0 or adding it to the url